api/types/container/container.go

```diff
@@ -121,19 +121,20 @@ type State struct {
 // Summary contains response of Engine API:
 // GET "/containers/json"
 type Summary struct {
-	ID         string `json:"Id"`
-	Names      []string
-	Image      string
-	ImageID    string
-	Command    string
-	Created    int64
-	Ports      []Port
-	SizeRw     int64 `json:",omitempty"`
-	SizeRootFs int64 `json:",omitempty"`
-	Labels     map[string]string
-	State      string
-	Status     string
-	HostConfig struct {
+	ID                      string `json:"Id"`
+	Names                   []string
+	Image                   string
+	ImageID                 string
+	ImageManifestDescriptor *ocispec.Descriptor `json:",omitempty"`
```

Comment:
oh! Before I blame myself; we've been very lax on specifying the _intended_ casing of fields in JSON (depending on "whatever Go makes of it); this has resulted in inconsistencies where the Swagger documented the wrong casing (the `Id` one is a good example of that, where we had to later adjust the case to what it happened to be). Go itself treats JSON case-insensitive, but not all languages do (and ISTR Go is working on a "v2" JSON package to allow case-sensitivity).

Probably good to add an explicit `json:"ImageManifestDescriptor,omitempty"` (at least for new fields; we should also tackle existing fields at some point)